server/src/config.js

```diff
@@ -25,7 +25,30 @@ module.exports = Object.freeze({
       {
         kind: 'video',
         mimeType: 'video/VP8',
-        clockRate: 90000
+        clockRate: 90000,
+        parameters: {
+          'x-google-start-bitrate': 1000
+        }
+      },
+      {
+        kind: 'video',
+        mimeType: 'video/VP9',
+        clockRate: 90000,
+        parameters: {
+          'profile-id': 2,
+          'x-google-start-bitrate': 1000
+        }
+      },
+      {
+        kind: 'video',
+        mimeType: 'video/H264',
+        clockRate: 90000,
+        parameters: {
+          'packetization-mode': 1,
+          'profile-level-id': '4d0032',
```

Comment:
Do the majority of browsers support profile level 4d0032?

Reply:
These settings are copied from mediasoup's configure